src/main/java/nextstep/courses/domain/Size.java

```diff
@@ -0,0 +1,41 @@
+package nextstep.courses.domain;
+
+import javax.swing.*;
+
+public class Size {
+
+    private static final int DEFAULT_WIDTH = 300;
+    private static final int DEFAULT_HEIGHT = 200;
+
+    private final int width;
+    private final int height;
+
+    public Size(int width, int height) {
+        this.width = width;
+        this.height = height;
+        validate();
+    }
+
+    private void validate() {
+        validateSize();
+        validateRatio();
+    }
+
+    private void validateSize() {
+        if(width < DEFAULT_WIDTH && height < DEFAULT_HEIGHT){
```

Comment:
width 또는 height 둘 중 하나라도 최소 픽셀을 충족하지 못하는 경우에도 예외를 던져야 하지 않을까요?
```suggestion
        if (width < DEFAULT_WIDTH || height < DEFAULT_HEIGHT) {
```